sqm-scripts/patches/000-OverTheBox-SQM-run.patch

```diff
@@ -1,162 +1,74 @@
-diff --git a/luci/sqm-cbi.lua b/luci/sqm-cbi.lua
-index 9b8af51..fd5dd53 100644
---- a/luci/sqm-cbi.lua
-+++ b/luci/sqm-cbi.lua
-@@ -13,6 +13,7 @@ You may obtain a copy of the License at
- $Id$
- ]]--
- 
-+local uci = luci.model.uci.cursor()
- local wa = require "luci.tools.webadmin"
- local fs = require "nixio.fs"
- local net = require "luci.model.network".init()
-@@ -55,17 +56,48 @@ function e.write(self, section, value)
- 	end
- 	return Flag.write(self, section, value)
- end
---- TODO: inform the user what we just did...
- 
--n = s:taboption("tab_basic", ListValue, "interface", translate("Interface name"))
-+-- TODO: inform the user what we just did...
-+n = s:taboption("tab_basic", ListValue, "_interface_all", translate("Interface name"))
- -- sm lifted from luci-app-wol, the original implementation failed to show pppoe-ge00 type interface names
- for _, iface in ipairs(ifaces) do
- 	if not (iface == "lo" or iface:match("^ifb.*")) then
--		n:value(iface)
-+		local multipath = uci:get("network", iface, "multipath")
```

Comment:
I've seen this code before [here](https://github.com/ovh/overthebox-feeds/blob/a2dac6d1eecdc1324b19d7cda318c3252ef3b7bf/luci-mod-admin-full/luasrc/controller/admin/status.lua#L88)
Wouldn't it be better to create a shared function ?